src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.html

```diff
@@ -39,6 +43,23 @@ <h3 class="header-start">
             </div>
         </div>
 
+        <ng-template #clearConfirmModal let-modal>
+            <div class="modal-header">
+                <h4 class="modal-title">
+                    <span>{{ 'artemisApp.exerciseChatbot.clearSession.title' | artemisTranslate }}</span>
+                </h4>
+                <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
+            </div>
+            <div class="modal-body">
+                <p>{{ 'artemisApp.exerciseChatbot.clearSession.text' | artemisTranslate }}</p>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-danger" (click)="modal.close('confirm')">
+                    <span>{{ 'artemisApp.exerciseChatbot.clearSession.submit' | artemisTranslate }}</span>
+                </button>
```

Comment:
Other modals also have a cancel button in Artemis

Reply:
https://github.com/ls1intum/Artemis/blob/cbcee52d843df6a9d31f891df207cf16cd08e04e/src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html#L27-L55
https://github.com/ls1intum/Artemis/blob/cbcee52d843df6a9d31f891df207cf16cd08e04e/src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html#L144-L168
https://github.com/ls1intum/Artemis/blob/cbcee52d843df6a9d31f891df207cf16cd08e04e/src/main/webapp/app/shared/components/course-exam-archive-button/course-exam-archive-button.component.html#L23-L63

I think most of them only have confirm button at the footer